<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineCalibration</class>
 <widget class="QDialog" name="MachineCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1888</width>
    <height>866</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_V1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阀一电机：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阀二电机：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>阀三电机：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刺破电机：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_door">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仓门电机：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_push">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注射电机：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v1_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v1_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v1_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V1_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_V1_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>140</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v1_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>140</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v1_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>140</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v2_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v2_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v2_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V2_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>240</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_V2_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>220</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v2_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>220</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v2_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>220</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v3_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v3_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>310</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V3_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v3_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v3_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>310</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_V3_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>310</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v3_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v4_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v4_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>410</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_V4_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v4_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>410</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v4_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>410</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_V4_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>410</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_v4_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>500</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_door_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>500</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_door_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>500</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_push_soft">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_push_save">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>590</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_push_data">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>610</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_push_work">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>590</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工装位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_push_set">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>590</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_push_data">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>590</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_push_hard">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>590</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_door_step">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仓门电机：  步数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_door_step">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>710</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gear">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>710</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>档位：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_gear">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>710</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_in">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>710</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>向内</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_door_out">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>710</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>向外</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_machine_back">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>740</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
